modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/common/Constants.java

```diff
@@ -24,140 +24,185 @@
+    public static final String CONTEXT_WAP_PROVISIONING_FILE = "WAP_PROVISIONING_FILE";
+    public static final String WINDOWS_PLUGIN_PROPERTIES = "WINDOWS_PLUGIN_PROPERTIES";
+
+    //Message handler constants
+    public static final String CONTENT_LENGTH = "Content-Length";
+    public static final String SECURITY = "Security";
+
+    //Web services media types
+    public static final String SYNCML_MEDIA_TYPE = "application/vnd.syncml.dm+xml;charset=utf-8";
+
+    /**
+     * Discovery service related other constants
+     */
+    public final class Discovery {
+        private Discovery() { throw new AssertionError(); }
+
+        public static final String ENROLL_SUBDOMAIN = "https://EnterpriseEnrollment.";
+        public static final String CERTIFICATE_ENROLLMENT_POLICY_SERVICE_URL =
+                "/ENROLLMENTSERVER/PolicyEnrollmentWebservice" +
+                ".svc";
+        public static final String CERTIFICATE_ENROLLMENT_SERVICE_URL =
+                "/ENROLLMENTSERVER/DeviceEnrollmentWebservice" +
+                ".svc";
+        public static final String ONPREMISE_CERTIFICATE_ENROLLMENT_POLICY =
+                "/ENROLLMENTSERVER/ONPREMISE/" +
+                "PolicyEnrollmentWebservice.svc";
+        public static final String ONPREMISE_CERTIFICATE_ENROLLMENT_SERVICE_URL =
+                "/ENROLLMENTSERVER/ONPREMISE/DeviceEnrollmentWebservice.svc";
+        public static final String WAB_URL = "/mdm/enrollments/windows/login-agent";
+        //public static final String WAB_URL = "/mdm/enrollment
+
+    }
+
+    /**
+     * Certificate enrolment policy service related constants
+     */
+    public final class CertificateEnrolmentPolicy {
+        private CertificateEnrolmentPolicy() { throw new AssertionError(); }
+
+        public static final int MINIMAL_KEY_LENGTH = 2048;
+        public static final int POLICY_SCHEMA = 3;
+        public static final int HASH_ALGORITHM_OID_REFERENCE = 0;
+        public static final int OID_REFERENCE = 0;
+        public static final String OID = "1.3.14.3.2.29";
+        public static final String OID_DEFAULT_NAME = "szOID_OIWSEC_sha1RSASign";
+        public static final int OID_GROUP = 1;
+        public static final int OID_REFERENCE_ID = 0;
+    }
+
+    /**
+     * Certificate enrollment Service related constants
+     */
+    public final class CertificateEnrolment {
+        private CertificateEnrolment () { throw new AssertionError(); }
+
+        public static final String TOKEN_TYPE =
+                "http://schemas.microsoft.com/5.0.0" +
+                ".0/ConfigurationManager/Enrollment/DeviceEnrollmentToken";
+        public static final String PARM = "parm";
+        public static final String TYPE = "type";
+        public static final String VALUE = "value";
+        public static final String VALUE_TYPE =
+                "http://schemas.microsoft.com/5.0.0" +
+                ".0/ConfigurationManager/Enrollment/DeviceEnrollmentProvisionDoc";
```

Comment:
Inappropriate line break of the string. better keep the total version number in one line